Cubical/HITs/Rationals/HITQ/Base.agda

```diff
@@ -0,0 +1,36 @@
+{-# OPTIONS --cubical --safe #-}
+module Cubical.HITs.Rationals.HITQ.Base where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Function
+open import Cubical.Relation.Nullary
+
+open import Cubical.HITs.Ints.QuoInt
+
+open import Cubical.Data.Nat as ℕ hiding (_*_)
+open import Cubical.Data.NatPlusOne
+open import Cubical.Data.Sigma
+
+
+-- ℚ as a higher inductive type
```

Comment:
It would be great to add an issue about proving that this is equivalent to the other representation(s)!

Reply:
Great, will do once this is merged!